### PR TITLE
Allow spaces on binary path

### PR DESCRIPTION
**Pull request by piranna**
This fix issue #320

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 node_modules
 .test
-.idea
\ No newline at end of file
+.idea
+phantom.js
```
Comment (amir20) [inline .gitignore]: huh? Why did you add these? They are not automatically compiled any where. So did I miss a fix some where else?

File: package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
   },
   "license": "MIT",
   "scripts": {
+    "prepublish": "cake build",
```
Comment (amir20) [inline package.json]: Oh wait, there is the fix.

**Comment by amir20** [comment]
build is not working. I am going to try it out locally first though

**Comment by amir20** [comment]
@piranna running the tests again worked. But i don't have access to your repo. I could fix it really quick locally but I thought it would be easier if you do it. Can you rebase with master and shoot me a note when you do? I can merge.

**Comment by amir20** [comment]
Actually, you know what. I'll just rebase and push since the tests pass.

**Comment by amir20** [comment]
Closing in favor of #325

**Comment by piranna** [comment]
Thank you for the quick fix! :-D

**Comment by amir20** [comment]
@piranna can you look at the #326, I can't look while at work. But it looks like your commit broke `shim.js` and it is missing. I can't tell why.
